### ISSUE TITLE
Fails testing with zef

### ISSUE THREAD

**Issue by tbrowder** [frithnanth/perl6-Archive-Libarchive-Raw#10]
Debian 10, amd64
Rakudo 2021.12

**Comment by frithnanth** [comment]
Debian sid, amd64
Rakudo 2021.12
```
$ prove6
t/00-use.rakutest ................ ok 
t/01-version.rakutest ............ ok 
t/02-list.rakutest ............... ok 
t/03-extract.rakutest ............ ok 
t/04-archive.rakutest ............ ok 
t/05-archive-read-disk.rakutest .. ok  
t/99-meta.rakutest ............... ok 
All tests successful.
Files=7, Tests=120,  10 wallclock secs
Result: PASS

```
```
$ zef test .
===> Testing: Archive::Libarchive::Raw:ver<0.1.0>
===> Testing [OK] for Archive::Libarchive::Raw:ver<0.1.0>
```
Please post the output of your test, because I cannot reproduce the problem.

**Comment by tbrowder** [comment]
```
zef install Archive::Libarchive::Raw
===> Searching for: Archive::Libarchive::Raw
===> Updating fez mirror: https://360.zef.pm/
# Failed test 'version number'
# at t/01-version.rakutest line 10
# expected: '3004000'
#      got: '3003003'
# Failed test 'version string'
# at t/01-version.rakutest line 11
# expected: 'libarchive 3.4.0'
#      got: 'libarchive 3.3.3'
# Failed test 'version details'
# at t/01-version.rakutest line 12
# expected: 'libarchive 3.4.0 zlib/1.2.11 liblzma/5.2.4 bz2lib/1.0.8 liblz4/1.9.1 libzstd/1.4.3'
#      got: 'libarchive 3.3.3 zlib/1.2.11 liblzma/5.2.4 bz2lib/1.0.6 liblz4/1.8.3 libzstd/1.3.8'
# Failed test 'linked bzlib version'
# at t/01-version.rakutest line 17
# expected: '1.0.8, 13-Jul-2019'
#      got: '1.0.6, 6-Sept-2010'
# Failed test 'linked liblz4 version'
# at t/01-version.rakutest line 18
# expected: '1.9.1'
#      got: '1.8.3'
# Failed test 'linked libzstd version'
# at t/01-version.rakutest line 19
# expected: '1.4.3'
#      got: '1.3.8'
# You failed 6 tests of 8
===> Building: Archive::Libarchive::Raw:ver<0.1.0>
===> Building [OK] for Archive::Libarchive::Raw:ver<0.1.0>
===> Testing: Archive::Libarchive::Raw:ver<0.1.0>
===> Testing [FAIL]: Archive::Libarchive::Raw:ver<0.1.0>
===> Updated fez mirror: https://360.zef.pm/
===> Updating p6c mirror: https://raw.githubusercontent.com/ugexe/Perl6-ecosystems/master/p6c1.json
===> Updating cpan mirror: https://raw.githubusercontent.com/ugexe/Perl6-ecosystems/master/cpan1.json
===> Updated p6c mirror: https://raw.githubusercontent.com/ugexe/Perl6-ecosystems/master/p6c1.json
===> Updated cpan mirror: https://raw.githubusercontent.com/ugexe/Perl6-ecosystems/master/cpan1.json
Aborting due to test failure: Archive::Libarchive::Raw:ver<0.1.0> (use --force-test to override)
```

**Comment by tbrowder** [comment]
I just realized I have TEST_AUTHOR defined--let me try again...

**Comment by tbrowder** [comment]
It works fine, now--sorry for the noise. Happy New Year!
